### ISSUE TITLE
Perinatal health Risk Predictors Using Classification And Random Forest

### ISSUE THREAD

**Issue by vivekvardhan2810** [SrijanShovit/HealthLearning#149]
**Is your feature request related to a problem? Please describe.**

The Feature Has the potential to improve prenatal diagnosis of birth defects and outcomes in assisted reproductive technology too.

**Describe the solution you'd like**

I would like to develop a machine learning-based classification system for Perinatal health Risk Predictors.

Dataset Link : https://archive.ics.uci.edu/dataset/863/maternal+health+risk

Model : Train and compare various ML models, including Decision Tree, KNN, SVC, Random Forest, Logistic Regression, Bagging Classifier, AdaBoost Classifier, and Naive Bayes.

Data Preprocessing: This activity includes the following steps.

● Handling missing values
● Handling categorical data
● Handling outliers
● Scaling Techniques
● Splitting dataset into training and test set

Evaluation: Use GridSearchCV for hyperparameter tuning and evaluate model performance using confusion matrix and classification report, Choose the best-performing model based on accuracy and other performance metrics, typically Random Forest in this case.

**Describe alternatives you've considered**

No response.

**Additional context**

No response.

**Code of Conduct**

- [x] I agree to follow this project's Code of Conduct

**Comment by SrijanShovit** [comment]
Closing as not checked the repo and proposed same project with different name.

**Comment by vivekvardhan2810** [comment]
Why is my Issue closed
May I know the reason @SrijanShovit

**Comment by vivekvardhan2810** [comment]
Why did you Close this Issue Sir
As i am already working this issue
@SrijanShovit, may i know the reason
